Frameworks/QuartzCore/CALayer.mm

```diff
@@ -833,6 +831,8 @@ - (void)removeFromSuperlayer {
     [CATransaction _removeLayer:self];
 
     pSuper->priv->removeChild(self);
+
+    [pSuper setNeedsLayout];
     [self release];
```

Comment:
indeed strange. we will re-visit its usage pattern later though.

---
In reply to: [90150584](https://github.com/Microsoft/WinObjC/pull/1471#discussion_r90150584) [](ancestors = 90150584)